tests/contest/runtimetest/src/tests.rs

```diff
@@ -561,6 +561,201 @@ pub fn test_io_priority_class(spec: &Spec, io_priority_class: IOPriorityClass) {
+            parts.next()?; // skip address field
+            let policy_field = parts.next()?;
+            Some((policy_field, line))
+        })
+        .collect();
+
+    if policy_entries.is_empty() {
+        eprintln!("no parsable entries found in /proc/self/numa_maps");
+        return;
+    }
+
+    let fallback_entry = policy_entries[0];
+    let default_policy_field = fallback_entry.0;
+    let find_with_substring = |needle: &str| -> (&str, &str) {
+        policy_entries
+            .iter()
+            .copied()
+            .find(|(policy, _)| policy.contains(needle))
+            .unwrap_or(fallback_entry)
+    };
+
+    // Get auxiliary info from spec (nodes/flags)
+    let expected_nodes = expected_nodes_from_spec(spec);
+    let (nodes_is_empty, has_static_flag, has_relative_flag) = if let Some(p) = memory_policy {
+        let nodes_is_empty = p.nodes().as_ref().is_none_or(|n| n.trim().is_empty());
+        let mut has_static = false;
+        let mut has_relative = false;
+        if let Some(flags) = p.flags() {
+            for f in flags {
+                use oci_spec::runtime::MemoryPolicyFlagType::*;
+                match f {
+                    MpolFStaticNodes => has_static = true,
+                    MpolFRelativeNodes => has_relative = true,
+                    _ => {}
+                }
+            }
+        }
+        (nodes_is_empty, has_static, has_relative)
+    } else {
+        (true, false, false)
+    };
+
+    // Verify the policy matches expected mode
+    match expected_mode {
+        Some(MemoryPolicyModeType::MpolDefault) => {
+            let (policy_field, _) = find_with_substring("default");
+            if !policy_field.contains("default") {
+                eprintln!("expected default policy, but found: {}", policy_field);
+            }
+        }
+        Some(MemoryPolicyModeType::MpolInterleave) => {
+            let (policy_field, full_line) = find_with_substring("interleave");
+            if !policy_field.contains("interleave") {
+                eprintln!("expected interleave policy, but found: {}", policy_field);
+            }
+            if expected_nodes.as_ref().is_some_and(|v| v.contains(&0))
+                && !numa_maps_indicates_node0(policy_field)
+            {
+                eprintln!(
+                    "expected interleave including node0, but got: {}",
+                    full_line
+                );
+            }
+        }
+        Some(MemoryPolicyModeType::MpolBind) => {
+            let (policy_field, full_line) = find_with_substring("bind");
+            if !policy_field.contains("bind") {
+                eprintln!("expected bind policy, but found: {}", policy_field);
+            }
+            if has_static_flag && !policy_field.contains("static") {
+                eprintln!("expected bind static, but found: {}", policy_field);
+            }
+            if expected_nodes.as_ref().is_some_and(|v| v.contains(&0))
```

Comment:
I’d like it to support checking values other than 0, too.

example image
```
const EXPECT_NODE = 1;
if expected_nodes.as_ref().is_some_and(|v| v.contains(EXPECT_NODE))
 && !numa_maps_expected_node(policy_field) // check for EXPECT_NODE
```

I know we’ll specify only 0 on GitHub Actions, but I want to be able to specify 1 (etc.) locally and still run valid tests.(Supporting 0–1 ranges would be ideal, but not required.)